src/tooltip/test/tooltip.spec.js

```diff
@@ -277,6 +278,52 @@ describe('tooltip', function() {
 
   });
 
+  describe('with specified popup close delay', function () {
+
+    beforeEach(inject(function ($compile) {
+      scope.delay='1000';
+      elm = $compile(angular.element(
+        '<span tooltip="tooltip text" tooltip-popup-delay="1000"' +
+        'tooltip-popup-close-delay="{{delay}}">Selector Text</span>'
+      ))(scope);
+      elmScope = elm.scope();
+      tooltipScope = elmScope.$$childTail;
+      scope.$digest();
+    }));
+
+    it('should open after timeout and close after the close delay timeout', inject(function ($timeout) {
+
+      elm.trigger('mouseenter');
```

Comment:
It is correct that `.trigger` is currently used but it is from `jQuery`. The present tests have been written a while ago, but more recently [the Project Philosophy](https://github.com/angular-ui/bootstrap/wiki/Project-Philosophy) sets the goal to get away from the `jQuery` dependency. So for new PRs, it might be a worth contribution towards the new Philosophy to support this trend.

This is actually much more than *style*. There are [subtle differences between two methods](http://api.jquery.com/triggerHandler/). As `Angular` chose the other one, we might want to follow, unless there is a reason not to.

Finally, I totally see your point of following the present conventions, so feel free not to worry about it if you prefer. I may submit a separate PR in the future.